src/packages/v4/models/split.ts

```diff
@@ -0,0 +1,11 @@
+import { JBSplit, SplitPortion } from "juice-sdk-core";
+import { zeroAddress } from "viem";
+
+export const defaultSplit: JBSplit = {
```

Comment:
i think we can move defaultSplits to the spot where it's used, and remove this file. WDYT?

Reply:
Good idea, done